### DISCUSSION TITLE
Mix Raw Query and Prisma Queries / BUG when array used with IN parameter exceeds 999

### DISCUSSION THREAD

**Discussion by younes-alouani** [prisma/prisma#14841] [Q&A]
I have a model Post with two attributes Title and Body. 
I want to filter Posts using Custom Postgresql Full Text Search Raw Queries
Then filter results even more using Prisma Classic Queries. 

Query 1 : 
```ts
const result1 = await prisma.$queryRaw`SELECT id FROM Post to_tsvector('spanish', title) @@ to_tsquery('spanish', **'carro& de & carreras'**);`
```

Query 2 : 
```ts
const result2 = await prisma.post.findMany({ where: { tags: { some: { tag: { id: 1 } } } } })
```
How can I get intersection of query 1 and query 2  using prisma syntax ?

I thought about getting array of ids of posts of query 1 then adding a filter by id in array in query 2, but I read that there is an issue (#14539) concerning this solution if the array has more than 999 elements.

**Comment by austincrim** [comment]
Hey @younes-alouani 👋,

Due to the bug you referenced, the safest way forward is probably to use [`$queryRaw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw) with a subquery fetching the list of id's.

Feel free to leave a comment describing your use case on that issue. Any info is really helpful as our engineers look to solve it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
